mesa/experimental/jupyter_viz.py

```diff
@@ -6,11 +6,15 @@
 import solara
 from matplotlib.figure import Figure
 from matplotlib.ticker import MaxNLocator
+from solara.components.columns import Columns
+import sys
 
+# from matplotlib.pyplot import figure
 import mesa
 
 # Avoid interactive backend
 plt.switch_backend("agg")
+plt.rcParams["figure.figsize"] = (7, 7)
```

Comment:
You shouldn't set the figure size globally, because this affects both the space/grid and the plots. Needs to be more fine-grained. This comments applies to #1820 as well.

Reply:
Will resolve this by tomorrow, thanks for pointing this out

Reply:
I think I need an explanation on why this hasn't been acted on in 3 weeks.

Reply:
this has been removed

Reply:
I know. It took 3 weeks. What is the explanation?

Reply:
sometimes I just want to do a bunch of changes in a go. Just waited for more ideas/changes. maybe not the best thing.

Reply:
My comment was not on brainstorming a design. It was pointing out a mistake. Waiting for 3 weeks to have your ideas settled is not a sufficient justification. You haven't responded to comments in #1820. I interpreted it as my feedback not being taken seriously. Please change your workflow.

Reply:
Is #1821 not worth your time either?

Reply:
at the moment -- No.

Reply:
I'm aware of #1821, it's just that I feel it would be much easier to build upon when #1825 merged and stable.

Reply:
Same with #1820. It will become irrelevant after #1825 is merged. Scaling happens automatically.

Reply:
#1821 is simple to implement, orthogonal to your existing 2 PRs. Waiting for #1825 is not a sufficient justification for not working on it. If you don't want to do it, please say sooner. It is a needed feature that needs to be delivered soon. I will just do it.

#1820 is needed because this PR only handles the total space visualization with, and doesn't fix the overlapping markers.

I am appalled by your responses here so far. No replies on #1820, then suddenly you decided to not work on it for a reason that you think it is irrelevant, and which it is not justifiable either.

Reply:
I am very sorry for my comments and the lack of understanding of how you allocate time for your workflow, given that you are busy with classes as well. It was uncalled for. I haven't said earlier that you should have communicated your thoughts that #1820 is unnecessary. But it would be great next time to tell this and so we can discuss about it. Thank you for what you have done so far, and hope we can get this done by CSSSA.